diagnostic.md

```diff
@@ -36,29 +41,35 @@ Now, we want to get a single donut from the server. Write the `curl` request
 you'd use to retrieve a single donut, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts/69
 ```
 
 Write an AJAX request to retrieve a single donut from the server.
 
 ```js
-let getDonut = /* your answer here */;
-```
+let getDonuts = $.ajax({
+    url: http://www.example.com/donuts/69,
+    method: 'GET',
+  });
+  ```
 
 ## Delete a Single Resource
 
 Write the `curl` request you'd use to delete a single donut, using whatever ID
 you'd like.
 
 ```sh
-# your answer here
+curl --include --request DELETE http://www.example.com/donuts/69
 ```
 
 Write an AJAX request to delete a single donut from the server.
 
 ```js
-let deleteDonut = /* your answer here */;
-```
+let deleteDonut = $.ajax({
```

Comment:
Again, here. But ajax format wise yes. Don't forget to return the call for the data to be passed around. into various methods